benches/method.rs

```diff
@@ -0,0 +1,40 @@
+#![feature(test)]
+
+extern crate test;
+
+use http::method::Method;
+use test::Bencher;
+
+fn make_all_methods() -> Vec<Vec<u8>> {
+    vec![
+        b"OPTIONS".to_vec(),
+        b"GET".to_vec(),
+        b"POST".to_vec(),
+        b"PUT".to_vec(),
+        b"DELETE".to_vec(),
+        b"HEAD".to_vec(),
+        b"TRACE".to_vec(),
+        b"CONNECT".to_vec(),
+        b"PATCH".to_vec(),
+        b"CUSTOM_SHORT".to_vec(),
+        b"CUSTOM_LONG_METHOD".to_vec(),
+    ]
+}
+
+#[bench]
+fn method_easy(b: &mut Bencher) {
+    let name = b"Get";
```

Comment:
Should this be `GET`? With lowercase, it will parse as a "custom".

Reply:
yehh GET :) 
Fix